index.js

```diff
@@ -215,12 +214,22 @@ function removeOptionRow(id) {
 
 function removeOptionRowEventListener(e) {
 
-    removeOptionRow(e.target.getAttribute('data-id'))
+    removeOptionRow(e.target.getAttribute('data-id'));
     if (options.children.length == 0) {
         createInputRow()
     }
 }
 
+function checkEmptyRow() {
```

Comment:
Lijkt me niet de juiste naam voor deze functie, bij deze naam verwacht ik een Boolean terug en dat ik een parameter moet mee geven. Mss is findEmptyOptionRow beter, dat zegt dat het iets over een row gaat teruggeven en dat hij gaat zoeken in de options.